resonate/stores/remote.py

```diff
@@ -35,7 +39,7 @@ def __init__(
         self._auth = auth or ((os.getenv("RESONATE_USERNAME", ""), os.getenv("RESONATE_PASSWORD", "")) if "RESONATE_USERNAME" in os.environ else None)
         self._encoder = encoder or Base64Encoder()
         self._timeout = timeout
-        self._retry_policy = retry_policy or Constant(delay=1, max_retries=3)
+        self._retry_policy = retry_policy or Constant(delay=1, max_retries=sys.maxsize)
```

Comment:
Can you add a comment here saying that we will retry forever until we implement the drop task strategy?